### PR TITLE
DOC-9882 Update installation instructions on the Docs Site

### PR DESCRIPTION
**Pull request by RayOffiah**
https://issues.couchbase.com/browse/DOC-9882

### CODE REVIEW

**Review by osfameron — APPROVED** [review]
This "looks" great!
(as in, specifically, I haven't built it or tried out the instructions.
If you're looking for that kind of feedback, I could give it a go on Friday.
But approved to unblock if helpful for now.)

File: home/modules/contribute/pages/playbook.adoc [hunk 1]
```diff
@@ -31,18 +31,74 @@ You've already completed the steps on this page.
...
+
+. If there isn't a `branches` key on the line directly below the `url` key, add it and assign it the list of version branch names, except for the version branch your working branch is based on.
+In place of the remote version branch, use the value of `HEAD`.
+`HEAD` tells Antora to use your working branch.
+For example, if your working branch is based off of the docs-server release/5.5 branch, you'd assign the following values to `branches`:
++
+[source,yaml]
+----
+- url: ./docs-server
+  branches: [release/4.*, release/5.0, release/5.1, HEAD]
```
Comment (RichardSmedley) [inline home/modules/contribute/pages/playbook.adoc:59]: If we put 7.0, 71., 6.*
it means ppl can c+p this if they prefer

File: home/modules/contribute/pages/playbook.adoc [hunk 2]
```diff
@@ -31,18 +31,74 @@ You've already completed the steps on this page.
...
+[#lightweight-local-playbook]
+[sidebar]
+.Lightweight Local Playbook
+****
+
+You may have to iterate over amending the Antora playbook file a number of times, commenting out repos to which you do not have access.
+An easier approach could be starting with an intentionally minimalist playbook, and only adding the repos that you're planning to work on.
+This will have to include the `docs-server` repo which, amongst other things, contains the site navigation.
+
+For example, to work on local copies of our nine SDK docs repos, and the Server docs, and also include couchbase-cli in your local docs build, take a look at our
```
Comment (RichardSmedley) [inline home/modules/contribute/pages/playbook.adoc:98]: ten SDKs, Kotlin is now GA  ;)

**Comment by RichardSmedley** [comment]
Couple of minor comments, otherwise LGTM. :+1:

File: home/modules/contribute/pages/playbook.adoc (hunk 1 above)
Comment (RayOffiah) [reply home/modules/contribute/pages/playbook.adoc:59]: Yep, good idea.

File: home/modules/contribute/pages/playbook.adoc (hunk 2 above)
Comment (RayOffiah) [reply home/modules/contribute/pages/playbook.adoc:98]: Fixed!

**Comment by RayOffiah** [comment]
> This "looks" great! (as in, specifically, I haven't built it or tried out the instructions. If you're looking for that kind of feedback, I could give it a go on Friday. But approved to unblock if helpful for now.)

If you could give it a quick run through that would be great. Thanks. 👍🏾

**Review by maria-robobug — APPROVED** [review]
👍 LGTM, I haven't had a chance to run through it myself, but we can always raise a PR later if any of us find issues.

**Pull request merged**